src/VocabularyManager.Core/Interfaces/IVocabularyStoreManager.cs

```diff
@@ -0,0 +1,14 @@
+﻿using VocabularyManager.Core.Entities;
+
+namespace VocabularyManager.Core.Interfaces
+{
+    public interface IVocabularyStoreManager
```

Comment:
most of the times interfaces should be in application layer (in your case usecases)